egs/csj/s5/RESULTS

```diff
@@ -1,134 +1,212 @@
-## These are results using the third edition of CSJ.
-for eval_num in `seq 3`; do echo "=== evaluation set $eval_num ===" ;\
- for x in exp/{tri,dnn}*/decode_eval${eval_num}*; do [ -d $x ] && grep WER $x/wer_* | utils/best_wer.sh; done ; done
-
-## Results of using training set that contains "academic" and "other" speech data (default).
-## If you want to use "simulated public speaking" and "dialog" data, you should check the following script [local/csj_data_prep.sh line 44].
+##### These experiments use the below line in csj_data_prep.sh
+## cat $CSJ/*/{A,M}*/*-wav.list 2>/dev/null | sort > $dir/wav.flist # Using "Academic lecture" and "other" data
```

Comment:
Please change these comments to be consistent with how csj_data_prep.sh works now.

Reply:
ok